tutorials/scripting/gdscript/gdscript_basics.rst

```diff
@@ -1526,27 +1530,43 @@ See :ref:`doc_running_code_in_the_editor` for more information.
 Memory management
 ~~~~~~~~~~~~~~~~~
 
-If a class inherits from :ref:`class_Reference`, then instances will be
-freed when no longer in use. No garbage collector exists, just
-reference counting. By default, all classes that don't define
-inheritance extend **Reference**. If this is not desired, then a class
-must inherit :ref:`class_Object` manually and must call ``instance.free()``. To
-avoid reference cycles that can't be freed, a :ref:`class_WeakRef` function is
-provided for creating weak references. Here is an example:
+GDScript does not use garbage collection. Instead, the :ref:`class_RefCounted`
```

Comment:
Well, Godot does not have a garbage collector, that is right in the original. Godot _does_ have garbage collection though - reference counting _is_ a form of it.
So IMO something like "Godot implements reference counting to free resources that are no longer used, instead of a garbage collector." would be better.